tests/frontend/unit/DpBulkEdit.spec.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * (c) 2010-present DEMOS E-Partizipation GmbH.
+ *
+ * This file is part of the package demosplan,
+ * for more information see the license file.
+ *
+ * All rights reserved
+ */
+
+import $ from 'jquery'
+import DpBulkEditStatement from '@DpJs/components/statement/assessmentTable/DpBulkEditStatement'
+import shallowMountWithGlobalMocks from '@DpJs/VueConfigLocal'
+import Vuex from 'vuex'
+
+global.$ = $
+
+describe('DpBulkEditStatement', () => {
+  it('should be an object', () => {
+    expect(typeof DpBulkEditStatement).toBe('object')
+  })
+
+  it('should be named DpBulkEdit', () => {
+    expect(DpBulkEditStatement.name).toBe('DpBulkEditStatement')
+  })
+
+  let store
+  let actions
+  let getters
+
+  const stubs = {
+    DpTiptap: '<h3>DpLinkModal</h3>'
+  }
+
+  /*
```

Comment:
Does it make sense to keep this test file in the current state? We should at least remove the commented out code, and the remaining tests are not very useful, so maybe we should just remove the file for now?

Reply:
removed in 3fa65a02922197bd820d723212eec72f047df901